lab-yunjoo/app/component/app-gallery-item/app-gallery-item-directive.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const angular = require('angular');
+angular.module('demoApp')
+.directive('appGalleryItem', function(){
+  return {
+    restrict:'E',
+    replace: true,
+    scope: {
+      imgSrc:'@',
```

Comment:
Why are you passing these through w/ '@' in this directive and with '=' in other directives?